app/electron/runCmd.ts

```diff
@@ -132,6 +267,29 @@ export function handleRunCommand(
   if (!checkCommandConsent(eventData.command, mainWindow)) {
     return;
   }
+console.log('>>>>>>>>>>>>>>>>>>>>', eventData.command, eventData.args, eventData.options);
+  // Handle elevated command requests with a dedicated confirmation dialog
+  if (eventData.elevated) {
```

Comment:
this should be `eventData.options.elevated`

with this change it worked on windows, although no logs were passed to the frontend